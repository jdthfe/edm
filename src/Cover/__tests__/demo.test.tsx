import webDemoTest from '@tests/shared/demoTest';
webDemoTest('Cover');
