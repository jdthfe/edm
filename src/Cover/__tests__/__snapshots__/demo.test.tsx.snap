// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CoverDemo 1`] = `
<body>
  <div>
    <div
      class="edm-wingblank Cover edm-wingblank-m"
    >
      <h4>
        Cover
      </h4>
      <button
        class="edm-button edm-button-l edm-button-primary"
        role="button"
      >
        Cover
      </button>
      <div
        class="edm-Cover"
      />
      <div
        class="edm-whitespace edm-whitespace-m"
      />
      <button
        class="edm-button edm-button-l edm-button-primary"
        role="button"
      >
        Cover | transparent
      </button>
      <div
        class="edm-Cover edm-Cover-transparent"
      />
      <div
        class="edm-whitespace edm-whitespace-m"
      />
      <h4>
        Cover.Transition
      </h4>
      <button
        class="edm-button edm-button-l edm-button-primary"
        role="button"
      >
        Cover.Transition | Portal
      </button>
    </div>
  </div>
  <div
    class="edm-Cover edm-fade-entry-done edm-fade"
  />
</body>
`;

exports[`renders Cover correctly 1`] = `
<div>
  <div
    class="edm-wingblank Cover edm-wingblank-m"
  >
    <h4>
      Cover
    </h4>
    <button
      class="edm-button edm-button-l edm-button-primary"
      role="button"
    >
      Cover
    </button>
    <div
      class="edm-whitespace edm-whitespace-m"
    />
    <button
      class="edm-button edm-button-l edm-button-primary"
      role="button"
    >
      Cover | transparent
    </button>
    <div
      class="edm-whitespace edm-whitespace-m"
    />
    <h4>
      Cover.Transition
    </h4>
    <button
      class="edm-button edm-button-l edm-button-primary"
      role="button"
    >
      Cover.Transition | Portal
    </button>
  </div>
</div>
`;
